extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -2218,10 +2218,20 @@
 				"&pageid=" + page);
 		};
 
+		_t.subscriberError = function() {
+			var
+				notify = renderTemplate(o.notifyTemplateSseLimit, {});
+
+			// Failback to long polling mode
+			smarthome.UI.showNotification(notify);
+			initSubscription(null);
+		};
+
 		ajax({
 			url: _t.subscribeRequestURL,
 			type: "POST",
-			callback: _t.startSubscriber
+			callback: _t.startSubscriber,
+			error: _t.subscriberError
```

Comment:
It seems that the "limit reached" message would be displayed even when the actual error isn't related to subscription limit, so maybe it's worth to check status code too.

Reply:
This is the unique error defined by the REST API.

Reply:
@resetnow : I have now updated the Basic UI part in order to display a more generic message notifying th euser that the SSE subscription failed leading to the fallback to the long polling mode.